test/test-component.js

```diff
@@ -110,6 +110,15 @@ describe('Simple Component instance', function() {
       expect(el.shadowRoot).to.be.ok;
     });
 
+    it('successfully finds the panel root', function() {
+      el.attachedCallback();
+      childEl = document.createElement('nested-child');
+      childEl.setAttribute('panel-parent', el.panelID);
+      el.el.appendChild(childEl);
+      childEl.attachedCallback();
+      expect(childEl.$panelRoot).to.equal(el);
```

Comment:
Yeah I guess I'm thinking of the high-level integration-test view where you have `el1 -> el2 -> el3` and if `el2` uses shadow dom, you want `el3` to keep finding `el1` as panel root no matter how our implementation changes.